### DISCUSSION TITLE
The school does not provide students with educational email, but Github ask me must  to provide it

### DISCUSSION THREAD

**Discussion by luluxiaoyu** [community/community#70541] [GitHub Education]
### Hi everyone,

The school does not provide students with educational email, but Github ask me must  to provide it
![image](https://github.com/community/community/assets/58876608/0753dfee-e60c-48b3-b232-f4a750a56609)
I tried to submit the national academic certification report and my university admission letter, but Github asked me to provide the edu email address

**Comment by itsdotnet** [comment]
You should enter your school email
This emails like if your school name **INHA** email would be _exmaple@inha.com_
Ask your school or university andminstration to get email address like this.

If school does not provide school email its impossible to get student pack
